alyssa_mckee/class_assignments/product.py

```diff
@@ -0,0 +1,38 @@
+class Product(object):
+	def __init__(self, price, item_name, weight, brand, cost, status="for sale"):
+		self.price = price
+		self.name = item_name
+		self.weight = weight
+		self.brand = brand
+		self.cost = cost
+		self.status = status
+	
+	def sell(self):
+		self.status = "sold"
+		return self
+	
+	def add_tax(self, tax):
+		return format(self.price + self.price*tax, '.2f')
+	
+	def Return(self, reason):
```

Comment:
Make sure to be consistent in your code with your styles. Instead of having some methods that are capitalized and the majority of others that are not. Pick a style and stick with it. Doing this will create better coding habits.

Reply:
yeah, I should have probably named that method something else. the assignment asked for a "return" method, I capitalized it because `return` is a keyword in python so I can't use the lowercased word as a method name. When I was doing the assignment I debated between naming it something slightly different from what was listed and follow the all lower case rule, or capitalize the R so I could use the same name as the assignment had listed. I ended up using the 'R', but I'll change it in my next commit

Reply:
The reasoning that you've made makes a lot of sense. I'm glad to see that was a cause for concern when you were debating how to write this, especially since it causes a syntax error with the lower case return method name. If I was to think of a method name I would use 'returned'.